### PR TITLE
OSD-7306 Adding a secondary router test for cloud ingress operator

### PR DESCRIPTION
**Pull request by bng0y**

### CODE REVIEW

File: pkg/e2e/operators/cloudingress/secondary_router.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+			deployment, err := operators.PollDeployment(h, "openshift-ingress", deploymentName)
+			Expect(err).ToNot(HaveOccurred(), "failed fetching deployment")
+			Expect(deployment).NotTo(BeNil(), "deployment is nil")
+
+			// wait 1 minute for all routers to start
+			time.Sleep(time.Duration(60) * time.Second)
+			Expect(deployment.Status.ReadyReplicas).To(BeNumerically("==", deployment.Status.Replicas))
+		})
+
+		ginkgo.It("should be deleted when removed to publishingstrategy", func() {
```
Comment (whereswaldon) [inline pkg/e2e/operators/cloudingress/secondary_router.go:54]:
```suggestion
		ginkgo.It("should be deleted when removed from publishingstrategy", func() {
```

**Review by whereswaldon — CHANGES_REQUESTED** [review]
A single typo in the test name, but that's all. Looking good!

File: pkg/e2e/operators/cloudingress/secondary_router.go (hunk 1 above)
Comment (bng0y) [reply pkg/e2e/operators/cloudingress/secondary_router.go:54]: Thanks. I will change that

Comment (bng0y) [reply pkg/e2e/operators/cloudingress/secondary_router.go]: solved

**Review by whereswaldon — APPROVED** [review]
/lgtm

**Pull request merged**